src/extension/features/budget/custom-average-budgeting/index.js

```diff
@@ -0,0 +1,111 @@
+import { controllerLookup, getEmberView } from 'toolkit/extension/utils/ember';
+import {
+  getEntityManager,
+  isCurrentMonthSelected,
+  isCurrentRouteBudgetPage,
+} from 'toolkit/extension/utils/ynab';
+import { formatCurrency } from 'toolkit/extension/utils/currency';
+import { Feature } from '../../feature';
+
+export class CustomAverageBudgeting extends Feature {
+  constructor() {
+    super();
+  }
+
+  shouldInvoke() {
+    return isCurrentRouteBudgetPage() && isCurrentMonthSelected();
```

Comment:
i think we should just still allow this feature to run on other months, we should just use `currentSelectedMonth` as the end date to our call to `isBetweenMonths`